addons/calendar/models/calendar_alarm.py

```diff
@@ -0,0 +1,294 @@
+                        break
+            else:
+                in_date_format = fields.Datetime.from_string(curEvent.start)
+                last_found = self.do_check_alarm_for_one_date(in_date_format, curEvent, max_delta, 0, after=last_notif_mail, notif=False, missing=True)
+                for alert in last_found:
+                    self.do_mail_reminder(alert)
+        IrConfigParameter.sudo().set_param('calendar.last_notif_mail', now)
+
+    @api.model
+    def get_next_notif(self):
+        ajax_check_every_seconds = 300
+        partner = self.env.user.partner_id
+        cal_last_notif_ack = self.env.user.calendar_last_notif_ack
+        all_notif = []
+
+        if not partner:
+            return []
+
+        all_events = self.get_next_potential_limit_alarm(ajax_check_every_seconds, partner_id=partner.id, mail=False)
+
+        for event in all_events:  # .values()
+            max_delta = all_events[event]['max_duration']
+            curEvent = self.env['calendar.event'].browse(event)
+            if curEvent.recurrency:
+                bFound = False
+                LastFound = False
+                for one_date in curEvent.get_recurrent_date_by_event():
+                    in_date_format = one_date.replace(tzinfo=None)
+                    LastFound = self.do_check_alarm_for_one_date(in_date_format, curEvent, max_delta, ajax_check_every_seconds, after=cal_last_notif_ack, mail=False)
+                    if LastFound:
+                        for alert in LastFound:
+                            all_notif.append(self.do_notif_reminder(alert))
+                        if not bFound:  # if it's the first alarm for this recurrent event
+                            bFound = True
+                    if bFound and not LastFound:  # if the precedent event had alarm but not this one, we can stop the search fot this event
+                        break
+            else:
+                in_date_format = fields.Datetime.from_string(curEvent.start)
+                LastFound = self.do_check_alarm_for_one_date(in_date_format, curEvent, max_delta, ajax_check_every_seconds, after=cal_last_notif_ack, mail=False)
+                if LastFound:
+                    for alert in LastFound:
+                        all_notif.append(self.do_notif_reminder(alert))
+        return all_notif
+
+    @api.multi
+    def do_mail_reminder(self, alert):
+        res = False
+
+        event = self.env['calendar.event'].browse(alert['event_id'])
+        alarm = self.env['calendar.alarm'].browse(alert['alarm_id'])
+
+        if alarm.alarm_type == 'email':
+            res = self.env['calendar.attendee']._send_mail_to_attendees(
+                [att.id for att in event.attendee_ids],
+                email_from=event.user_id.partner_id.email,
+                template_xmlid='calendar_template_meeting_reminder',
+                force=True
+            )
```

Comment:
I can clearly see here https://github.com/odoo/odoo/pull/9781/files#diff-5ea9ab1c17d33e8318c77148692090d0R116
_send_mail_to_attendees works on recordset then why not call it with recordset, you must get traceback here that _send_mail_to_attendees() got multiple values for keyword argument 'email_from'